src/data.py

```diff
@@ -0,0 +1,39 @@
+import os
+import cv2
+from typing import Tuple, List
+from sklearn.model_selection import train_test_split
+import pandas as pd
+
+
+def check_images(directory: str) -> Tuple[List[str], int, int]:
+    empty_or_corrupted: List[str] = []
+    num_correct: int = 0
+
+    for filename in os.listdir(directory):
+        file_path = os.path.join(directory, filename)
+        img = cv2.imread(file_path)
```

Comment:
Nie do końca rozumiem czemy `try` zniknęło zupełnie

Reply:
Z tego co doczytałem, jeśli cv2.imread nie otworzy pliku zwróci None, a nie wyjątek

Reply:
ok - to super